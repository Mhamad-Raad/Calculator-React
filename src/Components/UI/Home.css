.home {
  flex-direction: column;
  justify-content: flex-start;
  align-items: flex-start;
  padding: 20px 14px;
}

.home-header {
  font-size: 24px;
  font-weight: bolder;
  color: black;
}

.p1 {
  font-size: 16px;
  font-weight: normal;
  color: black;
}

.p2 {
  font-size: 16px;
  font-weight: normal;
  color: black;
}
