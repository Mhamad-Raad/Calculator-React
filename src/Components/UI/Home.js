import React from 'react';
import './Home.css';

export default function Home() {
  return (
    <div className="home">
      <h1 className="home-header">
        Welcome to the Home Page
      </h1>
      <p className="p1">
        Praesent at ultrices ligula. Suspendisse ut ex laoreet orci convallis rutrum at id leo.
        Morbi placerat tincidunt massa eget iaculis.
        Orci varius natoque penatibus et magnis dis parturient montes,
        nascetur ridiculus mus. Praesent in dolor vitae diam bibendum bibendum.
        Vestibulum feugiat justo libero, nec pellentesque turpis tincidunt at.
        In ac commodo libero, in vehicula felis. Duis ac nunc est.
      </p>

      <p className="p2">
        Donec auctor, nisl eget ultricies lacinia, nisl nunc aliquet nisl, nec
        tincidunt nunc nisl euismod nunc. Nulla facilisi. Donec auctor, nisl eget ultricies lacinia,
        nisl nunc aliquet nisl, nec tincidunt nunc nisl euismod nunc. Nulla facilisi.
      </p>
    </div>
  );
}
