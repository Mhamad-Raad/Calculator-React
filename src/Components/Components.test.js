import { fireEvent, render, screen } from '@testing-library/react';
import Home from '../Components/UI/Home';

describe('Testing Components', () => {
  test('Testing Homepage', () => {
    const home = render(<Home />);
    expect(home).toMatchSnapshot();
  });
});
