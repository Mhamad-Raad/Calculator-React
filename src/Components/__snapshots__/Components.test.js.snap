// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Components Testing Homepage 1`] = `[Function]`;
